db-scripts/insert-topic/README.md

```diff
@@ -0,0 +1,70 @@
+insert-topics
+==============
```

Comment:
Let's have some consistent header style please and structure - might want a markdown linter installed :)